lang/ui.fr.json

```diff
@@ -951,7 +947,7 @@
     "description": "Aria label for the large play button on the simulator board"
   },
   "simulator-stop": {
-    "defaultMessage": "Stop",
+    "defaultMessage": "Stop simulator",
```

Comment:
Should be changed in the ui.en.json file, not ui.fr.json.

Reply:
2d04f073f6ac3c7b0f5ce2edcc13ba4a013b6e13